tox.ini

```diff
@@ -158,7 +160,8 @@ commands_pre =
     pip freeze
 commands =
     pytest specutils \
-        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.entry_points"
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.entry_points" \
```

Comment:
```suggestion
```

Fixed by astropy/specutils#1024.

Reply:
This is the only change I really would like (simply to avoid yet another PR).

Reply:
Thanks for the fix. I removed this warning filter.